tests/DateTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace MLL\GraphQLScalars\Tests;
+
+use Generator;
+use MLL\GraphQLScalars\Date;
+use MLL\GraphQLScalars\DateScalar;
+
+final class DateTest extends DateScalarTest
+{
+    protected function scalarInstance(): DateScalar
+    {
+        return new Date();
+    }
+
+    public function validDates(): Generator
+    {
+        yield ['2020-04-20', '2020-04-20T00:00:00.000000+00:00'];
+    }
+
+    public function canonicalizeDates(): Generator
+    {
+        yield ['2020-4-20', '2020-04-20'];
```

Comment:
Not sure if this is actually a bad idea and we should improve upon Lighthouse.

Reply:
true, I'll remove this, aight?

We're testing the behaviour of php datetime anyway.

Reply:
yeah, you can remove it 👍